github/event_types.go

```diff
@@ -221,6 +221,14 @@ type EditChange struct {
 	Body *struct {
 		From *string `json:"from,omitempty"`
 	} `json:"body,omitempty"`
+	Base *struct {
+		Ref *struct {
+			From *string `json:"from,omitempty"`
+		} `json:"ref,omitempty"`
+		Sha *struct {
```

Comment:
```suggestion
		SHA *struct {
```

for idiomatic Go.

Reply:
https://github.com/google/go-github/pull/1619/commits/05744972613659beb36b076a03cf38f8446682fb